tests/ra_aid/agents/test_ciayn_agent.py

```
@@ -154,3 +155,49 @@ def test_trim_chat_history_both_limits():
     assert result[0] == initial_messages[0]
     assert result[1] == chat_history[-1]
+
+
+class TestFunctionCallValidation:
+    @pytest.mark.parametrize("test_input", [
+        "basic_func()", 
+        "func_with_arg(\"test\")",
+        "complex_func(1, \"two\", three)",
+        "nested_parens(func(\"test\"))",
+        "under_score()",
+        "with-dash()"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)